app/components/post_list/post/body/content/message_attachments/attachment_title.tsx

```diff
@@ -55,17 +57,14 @@ const AttachmentTitle = ({channelId, link, location, theme, value}: Props) => {
             <Markdown
                 channelId={channelId}
                 location={location}
-                isEdited={false}
-                isReplyPost={false}
                 disableHashtags={true}
                 disableAtMentions={true}
                 disableGallery={true}
-                autolinkedUrlSchemes={[]}
-                mentionKeys={[]}
+                mentionKeys={EMPTY_MENTION_KEYS}
+
                 theme={theme}
                 value={value}
                 baseTextStyle={style.title}
-                textStyles={{link: style.link}}
```

Comment:
Are we using the link style anywhere in this file or should we remove it as well?

Reply:
Yes, we are using when not using markdown.